packages/core/src/Constants/Hints.ts

```diff
@@ -0,0 +1,48 @@
+import { type SignPage } from "./Titles";
+
+type HintData = {
+  header?: string;
+  description?: string;
+};
+const basicOperationHeader = undefined;
+const basicOperationDescription = undefined;
+
+const finalizationConsequences =
+  "Finalized funds are returned to your balance, allowing you to spend them or stake them again to earn rewards. Delegation rewards for these funds stop once they are finalized.";
+const unstakingConsequences = `The unstaking process takes 4 cycles (~10 days) to unlock your staked funds, making them finalizable on request. You will receive delegation rewards for finalizable funds. ${finalizationConsequences}`;
+
+export const Hints: Record<SignPage, HintData> = {
+  TezSignPage: {
+    header: basicOperationHeader,
+    description: basicOperationDescription,
+  },
+  OriginationOperationSignPage: {
+    header: basicOperationHeader,
+    description: basicOperationDescription,
+  },
+  ContractCallSignPage: {
+    header: basicOperationHeader,
+    description: basicOperationDescription,
+  },
+  DelegationSignPage: {
+    header: "It takes 2 cycles (~6 days) to start receiving delegation rewards.",
```

Comment:
```suggestion
    header: "It takes at least 3 full cycles (~9 days) to start receiving delegation rewards. The actual delay will depend on your chosen baker payout strategy",
```
It takes at least 2 full cycles + we need to wait until the end of the cycles to get rewards:
- user delegates on cycle n.
- delegation is taken into account for cycle n+3, and delegation rewards are allocated to the baker during the cycle
- at the beginning of cycle n+4, baker can pay the rewards for the previous cycle,

Note that many bakers pay delegation rewards with further delays. This is off protocols